### ISSUE TITLE
Ruby

### ISSUE THREAD

**Issue by OlegKunitsyn** [isaqb-org/glossary#196]
`A wonderful programming language.`

1. Subjective.
2. Not mentioned in the CPSA-F_slides_wm.
3. Not mentioned in the curriculum-foundation-en.

The candidate for deletion.

**Comment by gernotstarke** [comment]
yes, you're right.  But allow us a little fun... we also have Pokemons wandering around in the curriculum :-)
